### PR TITLE
Upgrade to new tc-platform version 5.3.0-pre13

### PR DESCRIPTION
**Pull request by mathieucarbou**
Note: NmsAgent is now a permanent entity so it does not need to be created.

### CODE REVIEW

**Comment by mathieucarbou** [comment]
@henri-tremblay @ljacomet : if you can please have a look at the gradle file changes. This is to fix the gradle warning saying that `executable` is deprecated and will be removed in gradle 5.

**Comment by henri-tremblay** [comment]
I will let that to @ljacomet. Because I don't know :-) But can ask around if needed

**Comment by mathieucarbou** [comment]
test this please

**Comment by mathieucarbou** [comment]
build aborted ??? restarting...
test this please

**Comment by mathieucarbou** [comment]
@ljacomet @anthonydahanne : build has been aborted twice by someone or something. So I don't know what to do and I cannot not see anything relevant in the cloudbee logs

**Comment by ljacomet** [comment]
The cloudbees do not reflect the build log, to save space on successful builds. You need to look into the build artifacts and you will find a `gradle.log` that indicates that `org.ehcache.clustered.management.EhcacheManagerToStringTest` ends up never completing and causing the build to be killed after the timeout period has elapsed. Please have a look at that test.

**Comment by mathieucarbou** [comment]
I found which test is failing, i will fix it today. It just takes a lot of time to troubleshoot and fix because of gradle and the fact that it is not easily runnable from within the ide.

File: integration-test/build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ dependencies {
 }
 
 compileTestJava {
-  options.forkOptions.executable = Jvm.current().javacExecutable
+  options.forkOptions.javaHome = Jvm.current().javaHome
```
Comment (anthonydahanne) [inline integration-test/build.gradle]: what's the rational for that change ?

Comment (chrisdennis) [inline integration-test/build.gradle]:
```
The executable property on ForkOptions has been deprecated and is scheduled to be removed in Gradle 5.0. Please use javaHome instead.
        at build_4hmurz4widgtxhl5sjsel6ff1$_run_closure6$_closure22$_closure30.doCall(/Users/cdennis/src/ehcache3/build.gradle:235)
```

Comment (anthonydahanne) [inline integration-test/build.gradle]: okee, thanks

**Pull request merged**